### ISSUE TITLE
How to emulate an input tag?

### ISSUE THREAD

**Issue by paulyoung** [facebook/draft-js#657]
This may be considered a feature request but it's more of a question really.

As [discussed on Slack](https://draftjs.slack.com/archives/general/p1473720776000407), I'd like to make an `Editor` behave like an `input` tag.

I think a lot of that boils down to having the content continually scroll to right, and not wrapping.

I've tried [`draft-js-single-line-plugin`](https://github.com/icelab/draft-js-single-line-plugin) but the text still wraps on long lines.

I initially [tried creating a decorator](https://draftjs.slack.com/archives/general/p1473721610000410) which didn't quite work as I wanted, and later got a bit closer with the following CSS:

```css
.public-DraftEditor-content {
  height: 40px;
  overflow: scroll;
  padding-bottom: 10px; /* hide the scrollbar */
}

.public-DraftStyleDefault-block > span {
  white-space: nowrap;
}
```

This still isn't quite right and I don't feel good about styling these elements, but besides that it just seems like there must be a better way.

This could be considered a duplicated of #397, but hopefully this is a more comprehensive report.

Thanks in advance.

cc @makenosound @uiii @jivebot

**Comment by stopachka** [comment]
hey @paulyoung 

I would try making something like this work with two steps:

1. Disable inserting a new line on handleReturn
2. Update css to change the white-space property, 

Here's an example js fiddle: 

https://jsfiddle.net/m6z0xn4r/20/

**Comment by uiii** [comment]
Yes, for regular input it works, but try to insert some text with new lines.

**Comment by paulyoung** [comment]
Thanks @stopachka!

I'll give that a try in conjunction with something like the single line plugin.

**Comment by jisaacks** [comment]
I added a paste handler to remove new lines from pasted text, but I don't know how to get it to automatically scroll to the cursor after inserting the new text?

Updated fiddle:
https://jsfiddle.net/m6z0xn4r/151/

**Comment by OriR** [comment]
@jisaacks Not sure if that's still relevant for you but I managed to make it scroll on paste.
Here's the fiddle for it: https://jsfiddle.net/orir/m6z0xn4r/317/

@stopachka I used your solution, but this had unexpected consequences in IE.
[`overflow: hidden`](http://www.satzansatz.de/cssd/onhavinglayout.html#prop) causes something called `hasLayout` which is specific to IE that [adds resize handles and border to the element](https://stackoverflow.com/questions/21864047/how-to-disable-elements-selection-and-resizing-in-contenteditable-div).

I know this is a stretch, but is there any other way you might think of to achieve this also in IE?
